### ISSUE TITLE
Final segunda fecha.

### ISSUE THREAD

**Issue by mechibaron** [marcelogarberoglio/PDC_2024_02#10]
Jose Burgos 61525
Thomas Busso 62519
Maria Mercedes Baron 61187

ACLARACION: Jose Burgos no está anotado en SGA debido a que todavía no le subieron la nota del final de la correlativa de Sistemas Operativos. Es posible guardar esa nota hasta que se suba la nota de SO? 

PREGUNTA: Para entregar todavía no estamos habilitados en Campus. Se entrega igualmente por campus o por algún otro lado?

**Comment by marcelogarberoglio** [comment]
Sí, no hay problema, puedo guardar la nota. Apenas pueda que se anote y me avise.
No figuraban en Campus porque no habían informado el grupo. Recién creé el grupo y lo asocié a la entrega, fíjense si pueden subirlo en Campus
